### PR TITLE
Add Opulent-X BETA (main net)

### PR DESCRIPTION
**Pull request by BiTGarcia**

### CODE REVIEW

File: _data/chains/eip155-41500.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "networkId": 41500,
+  "nativeCurrency": {
+    "name": "Oxyn Gas",
+    "symbol": "OXYN",
+    "decimals": 18
+  },
+  "rpc": ["https://connect.opulent-x.com"],
+  "faucets": [],
+  "infoURL": "https://beta.opulent-x.com",
+
```
Comment (ligi) [inline _data/chains/eip155-41500.json:15]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]
Please remove this line

File: _data/chains/eip155-41500.json (hunk 1 above)
Comment (BiTGarcia) [reply _data/chains/eip155-41500.json:15]: copy. Thanks.

Comment (BiTGarcia) [reply _data/chains/eip155-41500.json:15]: I am done remove extra line but I got an error (Some checks were not successful)

Comment (ligi) [reply _data/chains/eip155-41500.json:15]: can you run the prettier?

Comment (BiTGarcia) [reply _data/chains/eip155-41500.json]: Done. It looks good. Thank you so much.

Comment (BiTGarcia) [reply _data/chains/eip155-41500.json]: Oh. I got still.

Comment (ligi) [reply _data/chains/eip155-41500.json]: do `npx prettier --write _data/*/*.json` then commit

Comment (BiTGarcia) [reply _data/chains/eip155-41500.json]: Done and committed.

Comment (ligi) [reply _data/chains/eip155-41500.json]: sure? I still see a diff after running the prettier:
```

--- a/_data/chains/eip155-18000.json
+++ b/_data/chains/eip155-18000.json
@@ -1,9 +1,7 @@
 {
   "name": "Frontier of Dreams Testnet",
   "chain": "Game Network",
-  "rpc": [
-    "https://rpc.fod.games/"
-  ],
+  "rpc": ["https://rpc.fod.games/"],
   "nativeCurrency": {
     "name": "ZKST",
     "symbol": "ZKST",
@@ -14,9 +12,11 @@
   "chainId": 18000,
   "networkId": 18000,
   "infoURL": "https://goexosphere.com",
-  "explorers": [{
-    "name": "Game Network",
-    "url": "https://explorer.fod.games",
-    "standard": "EIP3091"
-  }]
+  "explorers": [
+    {
+      "name": "Game Network",
+      "url": "https://explorer.fod.games",
+      "standard": "EIP3091"
+    }
+  ]
 }
```

Comment (BiTGarcia) [reply _data/chains/eip155-41500.json]: Opsssss sorry. I did, but I only committed my file.

Comment (BiTGarcia) [reply _data/chains/eip155-41500.json]: I have just ran prettier and committed all affected files.

Comment (BiTGarcia) [reply _data/chains/eip155-41500.json]: image

This is what I did.

**Pull request merged**